<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
<comment> Begin CVS Header </comment>
<comment>   $Source: /Volumes/Home/Users/shoops/cvs/copasi_dev/copasi/plotUI/HistoWidget.ui,v $ </comment>
<comment>   $Revision: 1.6 $ </comment>
<comment>   $Name:  $ </comment>
<comment>   $Author: shoops $ </comment>
<comment>   $Date: 2012/03/15 17:04:47 $ </comment>
<comment> End CVS Header </comment>

<comment> Copyright (C) 2012 by Pedro Mendes, Virginia Tech Intellectual </comment>
<comment> Properties, Inc., University of Heidelberg, and The University </comment>
<comment> of Manchester. </comment>
<comment> All rights reserved. </comment>

 <comment> All rights reserved. </comment>
 <class>HistoWidget</class>
 <widget class="QWidget" name="HistoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>219</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form2</string>
  </property>
  <layout class="QGridLayout">
   <item row="2" column="0" colspan="2">
    <widget class="Line" name="mpLine">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <layout class="QHBoxLayout">
     <item>
      <widget class="QCheckBox" name="mpCheckBefore">
       <property name="text">
        <string>before</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="mpCheckDuring">
       <property name="text">
        <string>during</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="mpCheckAfter">
       <property name="text">
        <string>after Task</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="mpSpacerCaptureData">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>160</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="mpLblHistogram">
     <property name="text">
      <string>&lt;h3&gt;Histogram&lt;/h3&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="mpLblTitle">
     <property name="text">
      <string>Title</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="mpLblCaptureData">
     <property name="text">
      <string>Capture Data</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLineEdit" name="mpEditVariable">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="mpBtnVariable">
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>select</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="1">
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLineEdit" name="mpEditIncrement"/>
     </item>
     <item>
      <spacer name="mpSpacer1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>31</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="mpEditTitle"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="mpLblIncrement">
     <property name="text">
      <string>Increment</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="mpLblVariable">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Variable</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <spacer name="mpSpacer2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>mpBtnVariable</sender>
   <signal>clicked()</signal>
   <receiver>HistoWidget</receiver>
   <slot>buttonPressedX()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
